webapp/portlet/src/main/webapp/people-list/components/PeopleCardFront.vue

```diff
@@ -258,7 +259,7 @@ export default {
       return this.profileActionExtensions.slice().filter(extension => extension.enabled(this.user));
     },
     canUseActionsMenu() {
-      return this.user && this.enabledProfileActionExtensions.length;
+      return this.user && this.user.username !== this.currentUserName  && this.enabledProfileActionExtensions.length;
```

Comment:
Why this ? If I'm manager of the space, I shouldn't be able to promote myself or revoke myself as redactor ?

Reply:
The option "Set as a redactor" must be deleted for space managers,

Reply:
If the authenticated user is a manager, he shouldn't be able to promote himself or any other manager as a redactor because all space managers will be by default and permanent redactors